### PR TITLE
A11y fixes

### PR DESCRIPTION
**Pull request by cahrens**
@staubina and @jlajoie Please review this PR as a companion to https://github.com/edx/edx-platform/pull/14486.

FYI @cptvitamin

### CODE REVIEW

File: wiki/templates/wiki/dir.html [hunk 1]
```diff
@@ -22,14 +22,14 @@
       {% trans "Add article" %}
     </a>
   </div>
-  <div class="pull-right">
-    {{ filter_form.query }}
+  <div class="pull-right input-prepend">
```
Comment (cahrens) [inline wiki/templates/wiki/dir.html:25]: Add ?w=1 to suppress whitespace differences (though note that then you won't see comments or be able to create them).

File: wiki/templates/wiki/dir.html [hunk 2]
```diff
@@ -22,14 +22,14 @@
...
+    {{ filter_form.query.label_tag }} {{ filter_form.query }}
+    {% if filter_query %}
```
Comment (cahrens) [inline wiki/templates/wiki/dir.html:27]: Previously "(clear)" was appearing to the left of the input field when it displayed.

image

File: wiki/templates/wiki/dir.html [hunk 3]
```diff
@@ -43,39 +43,35 @@
...
-      </td>
-      <td>
-        {{ urlpath.slug }}
-      </td>
-      <td>
-        {{ urlpath.article.current_revision.created|naturaltime }}
-      </td>
-    </tr>
-    {% empty%}
+{% if directory %}
```
Comment (cahrens) [inline wiki/templates/wiki/dir.html:52]: Only add the table if there are filter results.

**Comment by jlajoie** [comment]
Looks good to me :+1:

**Comment by staubina** [comment]
👍

File: wiki/templates/wiki/dir.html [hunk 4]
```diff
@@ -22,60 +22,62 @@
...
+  <table class="table table-striped">
     <tr>
-      <td colspan="100">
-        <em>{% trans "There are no articles in this level" %}</em>
-      </td>
+      <th>{% trans "Title" %}</th>
```
Comment (cptvitamin) [inline wiki/templates/wiki/dir.html]: @cahrens can you add ```scope="col"``` to each of these ```<th>``` elements?

**Review by cptvitamin — CHANGES_REQUESTED** [review]

File: wiki/templates/wiki/dir.html (hunk 4 above)
Comment (cahrens) [reply wiki/templates/wiki/dir.html]: Yep, will do so.

**Comment by cahrens** [comment]
@cptvitamin I did add scope="col". Going ahead and merging.

**Comment by cptvitamin** [comment]
👍

**Pull request merged**